doc/install.rst

```diff
@@ -145,7 +145,7 @@ Once the compilation is completed, install the Python package by running ``setup
 .. code-block:: bash
 
   cd ../python
-  python3 setup.py install --user --force
+  python3 setup.py install --user
```

Comment:
Removal of `--force` was discussed here https://github.com/neo-ai/neo-ai-dlr/pull/150